LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 ZuhRa HashiMi
+Copyright (c) 2022 Samiullah Bhadur
```

Comment:
- [ ] You seemed to have used the wrong name in your License, kindly add your own name